prod/westeurope/internal/api/functions_app_r3/function_app_slot_staging/terragrunt.hcl

```diff
@@ -88,8 +88,6 @@ inputs = {
 
```

Comment:
isn't this io-functions-app instead?

Reply:
@gunzip @balanza I fix it, but I will apply tomorrow..